awx/ui/client/src/app.start.js

```diff
@@ -49,6 +51,25 @@ function fetchLocaleStrings (callback) {
     request.fail(() => callback({ code: DEFAULT_LOCALE }));
 }
 
+function fetchConfig (callback) {
+    const request = $.ajax(`/api`);
```

Comment:
I'm seeing a CSP error and blank page when running the dev server (port 3000). Changing this to `$.ajax(`/api/`);` seems to fix it.